heron/examples/src/java/com/twitter/heron/examples/CustomGroupingTopology.java

```diff
@@ -18,17 +18,17 @@
 import java.util.List;
 import java.util.Map;
 
-import backtype.storm.Config;
-import backtype.storm.StormSubmitter;
-import backtype.storm.generated.GlobalStreamId;
-import backtype.storm.grouping.CustomStreamGrouping;
-import backtype.storm.task.OutputCollector;
-import backtype.storm.task.TopologyContext;
-import backtype.storm.task.WorkerTopologyContext;
-import backtype.storm.topology.OutputFieldsDeclarer;
-import backtype.storm.topology.TopologyBuilder;
-import backtype.storm.topology.base.BaseRichBolt;
-import backtype.storm.tuple.Tuple;
+import org.apache.storm.Config;
```

Comment:
+1. Please don't change these imports in this review. If we want to do this, we should have a review that is just about that, that people might chime in on. I know there are people with an opinion about this so you don't want potentially hide this change in a review with a title that's not related to it.